profiles/use.desc

```diff
@@ -30,6 +31,7 @@ bindist - Flag to enable or disable options for prebuilt (GRP) packages (eg. due
 blas - Add support for the virtual/blas numerical library
 bluetooth - Enable Bluetooth Support
 branding - Enable Gentoo specific branding
+brotli - Enable Brotli compression support
```

Comment:
(De)?

Reply:
Have it your way.

Reply:
I don't care about having it, the issue is you're being inconsistent, just strip it entirely or add the prefix everywhere.